guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
<blockquote>Most semantic segmentation model contains two parts, i.e an Encoder and a Decoder. The Encoder is responsible for the extracting out features from the image, the decoder is the one which predicts the class of the pixel at the end. A typical Encoder-Decoder for segmentation task looks like the below architecture.</blockquote><p> </p><p>Most semantic segmentation models consists of two parts, i.e an <strong>encoder </strong>and a <strong>decoder</strong>. The encoder is responsible for extracting features from the image, the decoder predicts the class of the pixel at the end. A typical <strong>encoder-decoder </strong>for segmentation task looks like the architecture shown below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>done.</p>

Reply:
<p>done.</p>